packages/ncids-js/package.json

```diff
@@ -11,7 +11,7 @@
 		"directory": "packages/ncids-js"
 	},
 	"publishConfig": {
-		"registry": "https://npm.pkg.github.com/@nciocpl",
+		"registry": "https://npm.pkg.github.com/",
```

Comment:
Why is this like this? 12 months ago this was originally set to be `https://npm.pkg.github.com/@nciocpl` and has been working ever since. So I have concerns with this change.